tests/test_instruments_zhinst.py

```diff
@@ -14,49 +24,50 @@ def create_offline_device_setup():
     Function returning a device setup
     """
 
-    descriptor = """\
-        instruments:
-            SHFQC:
-            - address: DEV12146
-              uid: device_shfqc
-            HDAWG:
-            - address: DEV8660
-              uid: device_hdawg
-            PQSC:
-            - address: DEV10055
-              uid: device_pqsc
-
-        connections:
-            device_shfqc:
-                - iq_signal: q0/drive_line
-                  ports: SGCHANNELS/0/OUTPUT
-                - iq_signal: q1/drive_line
-                  ports: SGCHANNELS/1/OUTPUT
-                - iq_signal: q0/measure_line
-                  ports: [QACHANNELS/0/OUTPUT]
-                - acquire_signal: q0/acquire_line
-                  ports: [QACHANNELS/0/INPUT]
-                - iq_signal: q1/measure_line
-                  ports: [QACHANNELS/0/OUTPUT]
-                - acquire_signal: q1/acquire_line
-                  ports: [QACHANNELS/0/INPUT]
-
-
-            device_hdawg:
-                - rf_signal: q0/flux_line
-                  ports: SIGOUTS/0
-                - rf_signal: q1/flux_line
-                  ports: SIGOUTS/1
-
-            device_pqsc:
-                - internal_clock_signal
-                - to: device_hdawg
-                  port: ZSYNCS/4
-                - to: device_shfqc
-                  port: ZSYNCS/0
-        """
-
-    device_setup = lo.DeviceSetup.from_descriptor(
+    # Instantiate Zh set of instruments[They work as one]
```

Comment:
Maybe this file can also by moved to zurich.py, but I am not sure. Keep it wherever it makes more sense.